render/detailed/connections.go

```diff
@@ -115,7 +110,7 @@ func incomingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 		TopologyID:  topologyID,
 		Label:       "Inbound",
 		Columns:     columnHeaders,
-		Connections: connectionRows(r, counts, isInternetNode(n)),
+		Connections: connectionRows(r, ns, counts, isInternetNode(n)),
```

Comment:
Safe enough, as they have all been pulled out of the rendered nodes at some point. It would be nice if that had better locality.